### PR TITLE
Added Celebratory Chests

### PR DESCRIPTION
**Pull request by kaincenteno**
Changed 2 items to AoE, and added script to use it.
Added 3 scripts for celebratory chest on each zone.
Added IDs for SpecialMessage for the Celebratory Chest.
Added function in Global for the Celebratory Chest stock

### CODE REVIEW

File: scripts/globals/items/joyous_serinette.lua [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+-- Item Effect: Change Music
+-----------------------------------------
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    return 0
+end
+
+function onItemUse(target)
+        target:ChangeMusic(0,214)   --  Eternal Oath
```
Comment (TeoTwawki) [inline scripts/globals/items/joyous_serinette.lua]: I'm not sure this is how retail accomplishes this

File: scripts/globals/shop.lua [hunk 2]
```diff
@@ -93,6 +93,30 @@ dsp.shop =
         dsp.shop.general(player, stock)
     end,
 
+    --[[ *******************************************************************************
+        send outpost shop dialog to player
```
Comment (TeoTwawki) [inline scripts/globals/shop.lua]: "outpost"?

File: scripts/globals/items/joyous_serinette.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/globals/items/joyous_serinette.lua]: do we have a packet cap of this item being used?

Comment (TeoTwawki) [reply scripts/globals/items/joyous_serinette.lua]: ps your indent is wonky

File: scripts/globals/shop.lua (hunk 2 above)
Comment (kaincenteno) [reply scripts/globals/shop.lua]: Sorry I forgot to fix the comment @TeoTwawki , don't hit me too hard lol

Comment (TeoTwawki) [reply scripts/globals/shop.lua]: _-threatens with a stick-_

Comment (kaincenteno) [reply scripts/globals/shop.lua]: fixed. Resolving conversation.

File: scripts/globals/items/joyous_serinette.lua (hunk 1 above)
Comment (kaincenteno) [reply scripts/globals/items/joyous_serinette.lua]: @TeoTwawki  fixed the indentention. Only thing it does on reail is just change the music as AoE.
No Dialogs saying that the music was changed.

Fix indentation. Let me know if there's an issue with the way this item is and I can just take it out from the PR and leave the other ones.

Comment (kaincenteno) [reply scripts/globals/items/joyous_serinette.lua]: @TeoTwawki  changed the animation to the correct one, shared the capture for both items being used on discord 😸

File: scripts/globals/items/joyous_serinette.lua [hunk 3]
```diff
@@ -0,0 +1,15 @@
+-----------------------------------------
+-- ID: 5425
+-- Item: Joyous Serinette
+-- Item Effect: Change Music
+-----------------------------------------
+require("scripts/globals/msg")
```
Comment (wrenffxi) [inline scripts/globals/items/joyous_serinette.lua]: unused require

File: scripts/globals/items/serene_serinette.lua [hunk 4]
```diff
@@ -0,0 +1,15 @@
+-----------------------------------------
+-- ID: 5424
+-- Item: Serene Serinette
+-- Item Effect: Change Music
+-----------------------------------------
+require("scripts/globals/msg")
```
Comment (wrenffxi) [inline scripts/globals/items/serene_serinette.lua]: unused require

File: scripts/globals/items/joyous_serinette.lua (hunk 3 above)
Comment (kaincenteno) [reply scripts/globals/items/joyous_serinette.lua]: removed, thank you @wrenffxi  =)

File: scripts/globals/items/serene_serinette.lua (hunk 4 above)
Comment (kaincenteno) [reply scripts/globals/items/serene_serinette.lua]: removed, thank you @wrenffxi  =)

**Comment by kaincenteno** [comment]
@cocosolos  thank u for the help with making item usable in alliance.

**Pull request merged**